frontend/components/Course/InstructorQueuePage/QueueFormFields.tsx

```diff
@@ -172,11 +172,6 @@ const QueueFormFields = ({
         }
         setInput({ ...input });
     };
-
-    const handlePinInputChange = (e, { checked }) => {
```

Comment:
Nit: Let's keep the `handlePinInputChange` function, but change the underlying implementation, so people reading this file have an easier time understanding.